### PR TITLE
remove github link on offline version of docs

### PR DESCRIPTION
**Pull request by benjaoming**
#4348

### CODE REVIEW

File: docs/kalite-theme/breadcrumbs.html [hunk 1]
```diff
@@ -13,11 +13,11 @@
             <a href="https://bitbucket.org/{{ bitbucket_user }}/{{ bitbucket_repo }}/src/{{ bitbucket_version}}{{ conf_py_path }}{{ pagename }}{{ source_suffix }}" class="fa fa-bitbucket"> Edit on Bitbucket</a>
           {% elif show_source and source_url_prefix %}
             <a href="{{ source_url_prefix }}{{ pagename }}{{ source_suffix }}">View page source</a>
-          {% elif show_source and has_source and sourcename %}
+          {% elif "" and show_source and has_source and sourcename %}
```
Comment (MCGallaspy) [inline docs/kalite-theme/breadcrumbs.html]: I'd say either remove these branches entirely or update this LOC: https://github.com/learningequality/ka-lite/issues/4348#issuecomment-182539950

**Comment by benjaoming** [comment]
@MCGallaspy please merge me

kitten

**Comment by MCGallaspy** [comment]
I had a comment -- no response?

**Comment by benjaoming** [comment]
Yes, the following commit?

**Comment by MCGallaspy** [comment]
I was referring to this: https://github.com/learningequality/ka-lite/pull/4831/files#r52510801

I meant to either remove the conditional block (which always evaluates to false, as far as I can tell) or update the branch to 0.16.x instead of removing it entirely. No reason to leave dead code lying around.

**Comment by benjaoming** [comment]
> I meant to either remove the conditional block (which always evaluates to false, as far as I can tell)

I left it on purpose and made it always evaluate to false on purpose, because viewing source and linking to Github are still interesting features and I've previously discussed the Github link as an interesting feature even for offline users, so they can be made aware of the possibility to contribute online. Hence, I left it disabled. It's a small issue, it's been fixed, it doesn't make the codebase any more or less complicated as it's just a bit of third-party template code that we've disabled.

Good point about the branch, I PR is wrongly targetted for develop and not 0.16.x

**Comment by MCGallaspy** [comment]
Okay, I didn't understand your reasoning but now I do.

**Pull request closed without merging**